### ISSUE TITLE
Players who try to enter either the nether or the end when they aren't enabled are invulnerable and cannot enter if they are enabled later

### ISSUE THREAD

**Issue by ghost** [Eluinhost/UHC#102]
All is fixed if they rejoin

Spigot 1.9.4, commit https://github.com/Eluinhost/UHC/commit/7847ecea867ca015d2a59dfa04126bc92ef5132c

**Comment by Eluinhost** [comment]
Hey, can you give us a list of plugins and your config and a list of instructions to trigger it? I'll take a look at it later when I get home

**Comment by ghost** [comment]
Only using the uhc plugin, default config.
1. Start the server
2. /uhc disable nether
3. Make a portal, try to enter
4. Done! You are now invulneable

**Comment by Eluinhost** [comment]
Can confirm, on clean install of latest 1.9.4 spigot with latest commit.

I'm thinking this is a spigot bug as being in creative the message 'Nether is disabled' is sent but when not it doesn't send the message and you end up invincible. There isn't anything wrong with the related disable code it's very simple. 

```
        if (isEnabled()) return;
        if (event.getTo() == null) return;

        if (event.getTo().getWorld().getEnvironment() == World.Environment.NETHER) {
            event.setCancelled(true);
            event.getPlayer().sendMessage(messages.getRaw("disabled message"));
        }
```

**Comment by Eluinhost** [comment]
I've opened the bug with Spigot https://hub.spigotmc.org/jira/browse/SPIGOT-2311 I'll keep this open until it's resolved

**Comment by Eluinhost** [comment]
Hi, this should be fixed as of Craftbukkit commit 042911fef46a2efd1fcc4266a01ede48e6e2948c, Spigot build rev 840
